src/utils.ts

```diff
@@ -266,3 +266,19 @@ export function searchLinkByRel(links: Link[], rel: string): Link[] {
   var regularExpression = new RegExp('\\b' + escapeRegExp(rel) + '\\b');
   return links.filter((l) => regularExpression.test(l.rel));
 }
+
+export function intlFormatNumber(lang: string, format: string, value: number) {
+  switch (format) {
+    case 'compact':
+      return new Intl.NumberFormat('en', {
+        //@ts-ignore
+        notation: 'compact',
+        //@ts-ignore
+        compactDisplay: 'short',
+      }).format(value);
+    case 'intl':
+      return new Intl.NumberFormat(lang === 'es' ? 'de' : lang).format(value);
```

Comment:
The reason that why I'm changing to german if language is spanish, is due to for NumberFormat, group separator starts from 10000 and not from 1000 when language is spanish.

Reply:
I don`t get the idea, can you give an example?

Reply:
Sure!
For `ESP`, if you do something like: 
`new Intl.NumberFormat('es').format(1000)` it will return `1000`
and if you do something like:
`new Intl.NumberFormat('es').format(10000)` it will return `10.000`
(it has to have more than four digits to apply the group separator)

But for `DE`, if you do something like:
`new Intl.NumberFormat('de').format(1000)` it will return `1.000`
(That is what we need)

I'm not sure, but I have read that is related with something with the RAE.

Reply:
Perhaps you could add a comment nearby explaining this bizarre issue.